packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ConnectAddressConfirmation.tsx

```diff
@@ -117,6 +122,9 @@ export const ConnectAddressConfirmation = () => {
                                         }
                                         size="small"
                                         isLoading={address.loading}
+                                        isDisabled={popupCall?.addresses.some(
```

Comment:
You can just use the `isLoading` variable

Reply:
👍 forcepushed